package.json

```diff
@@ -22,20 +22,23 @@
   ],
   "dependencies": {
     "apollo-client": "^1.4.2",
-    "babel-runtime": "^6.23.0"
+    "babel-preset-env": "^1.5.2",
```

Comment:
No, your dist files not bundled with there dependencies. See the screen shot, because your build with runtime-transfer.
<img width="1032" alt="screen shot 2017-06-22 at 12 44 29" src="https://user-images.githubusercontent.com/12751435/27419009-c64dad6a-5748-11e7-888c-2ec58a3b7df2.png">